### PR TITLE
Hyperledger Fabric WebAssembly chaincode Proposal

### PR DESCRIPTION
**Pull request by kleash**

### CODE REVIEW

**Comment by lehors** [comment]
+1!

File: labs/fabric-chaincode-wasm.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Lab Name
+Hyperledger Fabric WebAssembly chaincode
+
+# Short Description
+The project aims to integrate WebAssembly in hyperledger fabric, which will give chaincode developers a wider variety of languages to choose from when constructing their smart contracts.
```
Comment (tkuhrt) [inline labs/fabric-chaincode-wasm.md]: Capitalize Hyperledger Fabric.

File: labs/fabric-chaincode-wasm.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+More information [here](https://wiki.hyperledger.org/display/INTERN/Running+Web+Assembly+Smart+Contracts+in+Fabric)
+# Initial Committers
+Github IDs for the set of initial committers.
+- [MHBauer](https://github.com/MHBauer) - Morgan Bauer
+- [guoger](https://github.com/guoger) - Jay Guo
+- [swetharepakula](https://github.com/swetharepakula) - Swetha Repakula
+- [kleash](https://github.com/kleash) - Shubham Aggarwal
+
+# Sponsors
```
Comment (tkuhrt) [inline labs/fabric-chaincode-wasm.md]: For sponsors, could you include their role (e.g., maintainer, TSC member, etc.)?

Comment (tkuhrt) [inline labs/fabric-chaincode-wasm.md]: @swetharepakula @guoger @MHBauer : Can you confirm your sponsorship?

Comment (swetharepakula) [reply labs/fabric-chaincode-wasm.md]: Confirming

Comment (MHBauer) [reply labs/fabric-chaincode-wasm.md]: Yes.

Comment (tkuhrt) [reply labs/fabric-chaincode-wasm.md]: Thanks for confirming.

File: labs/fabric-chaincode-wasm.md (hunk 1 above)
Comment (kleash) [reply labs/fabric-chaincode-wasm.md]: Done

File: labs/fabric-chaincode-wasm.md (hunk 2 above)
Comment (kleash) [reply labs/fabric-chaincode-wasm.md]: Done

Comment (guoger) [reply labs/fabric-chaincode-wasm.md]: confirm!

File: labs/fabric-chaincode-wasm.md (hunk 1 above)
Comment (tkuhrt) [reply labs/fabric-chaincode-wasm.md]: I was actually talking about the "hyperledger fabric" on line 5. Can you revert the above change and fix on line 5 to reflect "Hyperledger Fabric".

Comment (kleash) [reply labs/fabric-chaincode-wasm.md]: Oh sorry, I misunderstood. Fixed it now

**Comment by tkuhrt** [comment]
LGTM +1

**Comment by kleash** [comment]
> @kleash could I ask you to squash this into one commit? I'm willing to merge at that point

@ryjones done. squashed all commits into one

Thanks

**Pull request merged**